@navikt/core/react/src/list/List.tsx

```diff
@@ -58,13 +58,19 @@ export const List = forwardRef<HTMLDivElement, ListProps>(
         <BodyLong
           as="div"
           {...rest}
-          size={size}
+          size={listSize}
           ref={ref}
           className={cl("navds-list", `navds-list--${listSize}`, className)}
         >
           {title && (
             <Heading
-              size={listSize === "medium" ? "small" : "xsmall"}
+              size={
+                listSize === "large"
+                  ? "medium"
+                  : listSize === "medium"
+                    ? "small"
+                    : "xsmall"
+              }
```

Comment:
What do we think about this? Could also do:
```
const headingSizeMap: Record<
  Exclude<ListProps["size"], undefined>,
  HeadingProps["size"]
> = {
  small: "xsmall",
  medium: "small",
  large: "medium",
};

(...)

<Heading size={headingSizeMap[listSize]} (...)
```

Reply:
Since it's only one place i think the current version works _ok_ without adding any extra complexity.

Reply:
Oops, didn't see this comment until now 😅 I change it b.c. Julian didn't like the nested ternary, and I think I agree.

Reply:
Agree that `headingSizeMap´ looks better 👍